apps/remix-ide/src/remixAppManager.js

```diff
@@ -390,7 +390,7 @@ class PluginLoader {
         localStorage.setItem('workspace', JSON.stringify(saved))
       },
       get: () => {
-        return JSON.parse(localStorage.getItem('workspace'))
+        return JSON.parse(localStorage.getItem('workspace') || '[]').filter(item => item !== 'dapp-draft')
```

Comment:
could you explain thy this change?

Reply:
I think I made a mistake here. I was not sure whether or not the dapp-draft plugin should be loaded in main panel, when I reload remix. 

I just checked other main panel plugins, they were loaded when I reload remix. So I guess it's fine for dapp-draft too.

I will recover this change soon. @yann300